specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json

```diff
@@ -0,0 +1,573 @@
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "WebAppServicePlanUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.AppServicePlanUpdated event.",
+      "type": "object",
+      "properties": {
+        "appServicePlanEventTypeDetail": {
+          "$ref": "#/definitions/AppServicePlanEventTypeDetail"
+        },
+        "sku": {
+          "description": "sku of app service plan.",
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "name of app service plan sku.",
+              "type": "string"
+            },
+            "Tier": {
+              "description": "tier of app service plan sku.",
+              "type": "string"
+            },
+            "Size": {
+              "description": "size of app service plan sku.",
+              "type": "string"
+            },
+            "Family": {
+              "description": "family of app service plan sku.",
+              "type": "string"
+            },
+            "Capacity": {
+              "description": "capacity of app service plan sku.",
+              "type": "string"
+            }
+          }
+        },
+        "name": {
+          "description": "name of the app service plan that had this event.",
+          "type": "string"
+        },
+        "clientRequestId": {
+          "description": "The client request id generated by the app service for the app service plan API operation that triggered this event.",
+          "type": "string"
+        },
+        "correlationRequestId": {
+          "description": "The correlation request id generated by the app service for the app service plan API operation that triggered this event.",
+          "type": "string"
+        },
+        "requestId": {
+          "description": "The request id generated by the app service for the app service plan API operation that triggered this event.",
+          "type": "string"
+        },
+        "address": {
+          "description": "HTTP request URL of this operation.",
+          "type": "string"
+        },
+        "verb": {
+          "description": "HTTP verb of this operation.",
+          "type": "string"
+        }
+      }
+    },
+    "AppEventTypeDetail": {
+      "description": "Detail of action on the app.",
+      "type": "object",
+      "properties": {
+        "action": {
+          "description": "Type of action of the operation.",
+          "type": "string",
+          "enum": [
+            "Restarted",
+            "Stopped",
+            "ChangedAppSettings",
+            "Started",
+            "Completed",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "AppAction",
+            "values": [
```

Comment:
Please add modelAsString: true.
This is the recommendation nowadays, the false by default is here jsut for legacy
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum